test/eth.accounts.signTransaction.js

```diff
@@ -551,7 +550,7 @@ describe("eth", function () {
                     testAccount.signTransaction(transaction)
                     .then(function (tx) {
                         assert.isObject(tx);
-                        assert.isString(tx.rawTransaction);
```

Comment:
This assertion should remain if this change is to be non-breaking...and same below...